packages/plugin-ext/src/main/browser/webview/pre/main.js

```diff
@@ -198,7 +198,7 @@ const acquireVsCodeApi = (function() {
         });
     };
 })();
-const acquireTheiaApi = acquireVsCodeApi;
+globalThis.acquireTheiaApi = acquireVsCodeApi;
```

Comment:
Isn't this a no-op?

Reply:
No, see reproduction steps of https://github.com/eclipse-theia/theia/issues/13374

Reply:
But we're already assigning to globalThis.actquireVsCodeApi further up in this file and here we're reading from the same variable (implicitly `acquireVSCodeApi === globalThis.acquireVsCodeApi) and assigning it a second time to `globalThis.acquireVsCodeApi`. What am I missing?

Reply:
Oh, that's what you mean. One is `acquireVsCodeApi` and the other one is `acquireTheiaApi`.

Reply:
If you're relying on me to read text with attention to detail, you're doomed ;-)